makefiles/Makefile.docs-java

```diff
@@ -66,6 +66,6 @@ next-gen-deploy:
 
 deploy-search-index: ## Update the search index for this branch
 	@echo "Building search index"
-	mut-index upload build/public -o ${PROJECT}-${GIT_BRANCH}.json -u ${PRODUCTION_URL}/${PROJECT} -g -s --exclude build/public/sdk/iOS
+	mut-index upload public -o ${PROJECT}-${GIT_BRANCH}.json -u ${PRODUCTION_URL}/${PROJECT} -s
```

Comment:
Did you mean to remove `-g`?

I don't think new search even uses global search, so I don't think it actually matters.

Reply:
Well, might as well make it consistent. How is this?